src/main/kotlin/com/atlassian/performance/tools/jiraactions/api/action/SetUpAction.kt

```diff
@@ -3,13 +3,27 @@ package com.atlassian.performance.tools.jiraactions.api.action
 import com.atlassian.performance.tools.jiraactions.api.SET_UP
 import com.atlassian.performance.tools.jiraactions.api.WebJira
 import com.atlassian.performance.tools.jiraactions.api.measure.ActionMeter
+import org.openqa.selenium.By
 
 class SetUpAction(
     private val jira: WebJira,
     private val meter: ActionMeter
 ) : Action {
     override fun run() {
         meter.measure(SET_UP) {
+
+            val popupDismissalLocators = listOf(
```

Comment:
or should that be in WebJira?

Reply:
or maybe there should be a LoginWithPopupCleanup action?

Reply:
The popup interactions themselves should be extracted to a [page object](https://github.com/SeleniumHQ/selenium/wiki/PageObjects), existing or new.
The call to `dismissPopups` can be included in the default login action. `LogInWithoutPopups` also sounds good.

`SetUpAction` is used once per test by an "admin user", which can be different from the individual VU users.

Reply:
It's not really a page object, because it's not a page... I think I'll add it to the login action, but run it only once per user.

Reply:
`SingleSelect` also isn't a page, but it represents a part of a page.

Reply:
Fair point. I can extract it. Should it be AnyPageWithPopups ? :-)